### PR TITLE
HW 1: David Whyman Bio

### PR DESCRIPTION
**Pull request by whymand**
After making a fork of the class hw 01 repository, I cloned it into my Rstudio on my computer; then, I made changes to the readme file. Periodically, I would commit the changes to my repository on the computer, then push the changes back up to my repository on GitHub. After looking at the guide linked into the homework instructions on the course website, I used the readme file to show me how to make text bold, italicized, and into headings and lists. The biggest problem I had was when I tried to insert an image into the readme file. I figured out how to upload the image of myself to GitHub, and then get a URL for the image, and then paste that into the readme. However, the problem I kept encountering was that the image would rotate 90 degrees when I uploaded it, so my face appeared sideways when I embedded it in the bio and pushed it to GitHub. I was eventually able to resolve this with more googling, as someone in a forum had encountered a similar problem and solved it by reducing the size of the jpeg file on their computer and then re-uploading the smaller-sized image. This solution worked for me as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,38 @@
...
+### About David
+I look at the world in an analytical way. Deconstructing complex issues and seeing how all the little parts interact is really cool. That's why I'm curious about social science research. Computer science and statistics provide useful tools to analyze these issues. Oh and by the way I love dogs. Especially **golden retrievers**.
+
+### David's Activities
+At the University of Chicago, I am involved in the Paul Douglas Institute, *The Citizen Bulletin*, and *The Maroon*. Check out [this article](https://www.chicagomaroon.com/article/2018/3/30/preckwinkle-talks-soda-tax-criminal-justice/) I wrote for *The Maroon*.
+
+
+
+<img style="float:left;transform: rotate(0deg); width:300px" src="https://github.com/whymand/myrepo/blob/master/Biopic.jpg" />
+
```
Comment (gmvelez) [inline README.md:29]: This is a bit of an odd addition here, and it sounds like you had a little bit of an issue getting the picture to insert.  One way to do it is save it on your computer under the folder that your homework project is in.  Then you include it almost just like you would with a link, but just putting the name of the document in the ().  So, for example, if you save the image as "Biopic.jpg" in the correct folder, then you can link it as:

[](Biopic.jpg)

When you commit and push to Git, the image will also be uploaded and will show up on your R markdown.

**Review by gmvelez — COMMENTED** [review]
Evaluation HW-01 for David Whyman
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |           |     x         |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |           |     x         |            |
| **Ease of access**          |      x     |              |            |

This is a good job since you cover an array of different formats, provide some good details, and have a detailed reflection.  It was also good that you committed as you did and seemed in the end to figure out via google how to get the image in (though I offered a suggestion of a more efficient way you could do it).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,38 @@
 # hw01
 Repository for homework 1
+------------------------------
+# A bio for David Whyman!
```
Comment (mobharmal) [inline README.md:4]: Good header!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,38 @@
...
+ 
+*David Whyman was born in Washington, DC on June 5, 1998.*
```
Comment (mobharmal) [inline README.md:6]: Nice italics!

File: README.md [hunk 4]
```diff
@@ -1,2 +1,38 @@
...
+
+### David's Education
```
Comment (mobharmal) [inline README.md:8]: Nice use of adjusting the size of the header!

File: README.md [hunk 5]
```diff
@@ -1,2 +1,38 @@
...
+David graduated from Walt Whitman High School in Bethesda Maryland. Now, David is a public policy major at the University of Chicago.
+
+### David's Hobbies!
```
Comment (mobharmal) [inline README.md:11]: Great list combined with header!

File: README.md [hunk 6]
```diff
@@ -1,2 +1,38 @@
...
+* Art
+   * Figure Drawing
+   * Landscape Painting
+* Horseback Riding 
+
+### About David
+I look at the world in an analytical way. Deconstructing complex issues and seeing how all the little parts interact is really cool. That's why I'm curious about social science research. Computer science and statistics provide useful tools to analyze these issues. Oh and by the way I love dogs. Especially **golden retrievers**.
+
+### David's Activities
+At the University of Chicago, I am involved in the Paul Douglas Institute, *The Citizen Bulletin*, and *The Maroon*. Check out [this article](https://www.chicagomaroon.com/article/2018/3/30/preckwinkle-talks-soda-tax-criminal-justice/) I wrote for *The Maroon*.
```
Comment (mobharmal) [inline README.md:24]: I like the way you embedded the link of the article here!

File: README.md [hunk 7]
```diff
@@ -1,2 +1,38 @@
...
+
+
+
+<img style="float:left;transform: rotate(0deg); width:300px" src="https://github.com/whymand/myrepo/blob/master/Biopic.jpg" />
```
Comment (mobharmal) [inline README.md:28]: Not sure why you added the image here at the end. Could have added the image at the top of the page by using the following Markdown syntax with the image saved to your computer:

**Review by mobharmal — COMMENTED** [review]
Evaluation HW-01 for David Whyman
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      yes        |            |
| **Coding strategy**         |           |        yes      |            |
| **Presentation**            |           |      yes        |            |
| **Achievement, creativity** |           |     yes         |            |
| **Ease of access**          |           |       yes       |            |

File looks good overall! The photo at the bottom is a bit awkward, but not a huge deal. Look to the individual comments above how you could have placed it closer to the top in case you needed help with that.

**Pull request closed without merging**